### PR TITLE
Renamed Primary Panes

### PR DESCRIPTION
**Pull request by pradeepgangwar**
### Summary of Changes

* Renamed Sources View tab to Sources 
* Renamed Outline View tab to Outline

### Screenshot

screenshot-2017-9-2 debugger

### CODE REVIEW

File: src/components/PrimaryPanes/index.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ class PrimaryPanes extends Component {
         onClick={() => this.showPane("outline")}
         key="outline-tab"
       >
-        Outline View
```
Comment (jasonLaster) [inline src/components/PrimaryPanes/index.js]: Oops, can we move these to `assets/panel/debugger.properties` w/ l10n

https://github.com/devtools-html/debugger.html/blob/master/docs/local-development.md#l10n

**Comment by pradeepgangwar** [comment]
@jasonLaster So i should write these in debugger.properties and fetch from there.

**Comment by jasonLaster** [comment]
Yep

**Comment by pradeepgangwar** [comment]
@jasonLaster Done.

File: assets/panel/debugger.properties [hunk 2]
```diff
@@ -237,6 +237,14 @@ breakpointMenuItem.disableAll.accesskey=k
 breakpointMenuItem.deleteAll=Remove all breakpoints
 breakpointMenuItem.deleteAll.accesskey=a
 
+# LOCALIZATION NOTE (panel.sources): Label displayed in the primary
+# pane. %S is replaced by the string sources.
+panel.sources=Sources
```
Comment (jasonLaster) [inline assets/panel/debugger.properties]: looks like already have this string: https://github.com/pradeepgangwar/debugger.html/blob/ed91f18bda935b6bc157413bbe71c6f427b3e1ab/assets/panel/debugger.properties#L434

File: assets/panel/debugger.properties [hunk 3]
```diff
@@ -237,6 +237,14 @@ breakpointMenuItem.disableAll.accesskey=k
...
+
+# LOCALIZATION NOTE (panel.outline): Label displayed in the primary
+# pane. %S is replaced by the string outline.
+panel.outline=Outline
```
Comment (jasonLaster) [inline assets/panel/debugger.properties]: lets move this one closer to the original Sources string

File: assets/panel/debugger.properties (hunk 2 above)
Comment (flodolo) [reply assets/panel/debugger.properties]: As I wrote a few days ago in Bugzilla, adding localization comments is not mandatory, they need to be helpful. In this case, try reading the localization comment and tell me if it matches the string (same for the next one) ;-)

@jasonLaster 
String reuse in different contexts should be avoided. Let's keep this separate from the one use from the header.

**Comment by pradeepgangwar** [comment]
@jasonLaster sorry i didn't see it was there. I have updated.

**Pull request merged**